.gitignore

```diff
@@ -1,3 +1,4 @@
 *__pycache__*
 *.npy
 deprecated
+agimus_controller_ros/agimus_controller_ros/agimus_controller_parameters.py
```

Comment:
What is this file?

Reply:
This file is self generated by ROS2 `generate_paramter_library`.
It's installed here for stupid reasons, it should always be in the build folder bu I did not had time to fix this properly.
So I added the .gitignore for now.
I will prevent people to actually commit in this file.